src/NServiceBus.Core/Pipeline/Incoming/TransportReceiveToPhysicalMessageProcessingConnector.cs

```diff
@@ -76,9 +75,10 @@ static void ConvertToPendingOperations(OutboxMessage deduplicationEntry, Pending
             }
         }
 
-        static List<TransportOperation> ConvertToOutboxOperations(IEnumerable<Transports.TransportOperation> operations)
+        static TransportOperation[] ConvertToOutboxOperations(Transports.TransportOperation[] operations)
```

Comment:
What am I supposed to do with thid comment @bording? We translate transport
stuff to outbox stuff

On Wednesday, July 6, 2016, Brandon Ording <notifications@github.com> wrote:

> In
> src/NServiceBus.Core/Pipeline/Incoming/TransportReceiveToPhysicalMessageProcessingConnector.cs
> <https://github.com/Particular/NServiceBus/pull/3903#discussion_r69658048>
> :
>
> > @@ -76,9 +75,10 @@ static void ConvertToPendingOperations(OutboxMessage deduplicationEntry, Pending
> >              }
> >          }
> >
> > -        static List<TransportOperation> ConvertToOutboxOperations(IEnumerable<Transports.TransportOperation> operations)
> > +        static TransportOperation[] ConvertToOutboxOperations(Transports.TransportOperation[] operations)
>
> It seems odd to me that we have to take in an array of TransportOperation
> and return a new array of equal length of newly allocated
> TransportOperations.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Particular/NServiceBus/pull/3903/files/9b3324353ab994bdf2836ce71186e54aed8cdac4#r69658048>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAKosuWiWcALNNh5WLF44wHdG9kdM-Zmks5qSu_kgaJpZM4JFK20>
> .
>